style.css

```diff
@@ -0,0 +1,823 @@
+    z-index: 2;
+    grid-column-gap: 31px;
+  }
+
+  .text {
+    width: 329px;
+    height: 338px;
+  }
+
+  .text__title {
+    font-size: 49px;
+    line-height: 50px;
+    letter-spacing: 1px;
+    width: 329px;
+  }
+
+  .text__container {
+    padding: 27px 0;
+  }
+
+  .text__subtitle {
+    font-size: 9px;
+    line-height: 13px;
+    letter-spacing: 1px;
+    margin-bottom: 24px;
+  }
+
+  .collection__next-text {
+    font-size: 8px;
+    padding: 7px 22px;
+    margin-right: 31px;
+    letter-spacing: 0;
+  }
+
+  .collection__other-text {
+    font-size: 8px;
+    letter-spacing: 0;
+    margin-right: 3px;
+  }
+
+  .collection__vector {
+    width: 27px;
+  }
+
+  .collection__scroll {
+    width: 17px;
+    height: 28px;
+    left: 46.5%;
+    top: 489px;
+  }
+
+  .collection__scroll-elem {
+    left: 47.3%;
+    top: 494.5px;
+    width: 2px;
+  }
+
+  .first__sidebar-social-inst,
+  .first__sidebar-social-vk,
+  .first__sidebar-social-fb {
+   	width: 25px;
+    height: 25px;
+  }
+
+  .first__sidebar-social {
+    width: 92px;
+    padding: 490px 42px 0;
+  }
+}
+
+@media (max-width: 750px) {
+  .header__language,
+  .header__vector,
+  .first__sidebar,
+  .first__collection,
+  .collection__scroll,
+  .collection__scroll-elem,
+  .img__img,
+  .text__subtitle,
+  .collection__next-text,
+  .collection__other-text,
+  .collection__vector{
+    display: none;
+  }
+
+  .first-container {
+    padding: 339px 0;
+    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
+    background-size: 750px;
+    background-position: center;
+  }
+
+  .header__inner {
+    width: 100%;
+    height: 140px;
+    background: #fff;
+    z-index: 2;
+  }
+
+  .header__menu {
+    background: #ededed;
+    z-index: 3;
+    height: 140px;
+    width: 140px;
+  }
+
+  .header__menu-item {
+    height: 4px;
+    width: 48.7px;
+    margin-bottom: 10px;
+  }
+
+  .header__logo p {
+    margin: 25px 21px;
+    color: black;
+    font-size: 60px;
+    letter-spacing: 4px;
+  }
+
+  .text__title {
+    position: inherit;
+    align-items: flex-end;
+    font-size: 98px;
+    line-height: 103px;
+    padding-left: 52px;
+    letter-spacing: 5px;
+  }
+
+  .text__container {
+    display: flex;
+    align-items: flex-end;
+    width: 100%;
+    max-width: 750px;
+    height: 945px;
+    letter-spacing: 5px;
+  }
+
+  .step {
+    display: flex;
+  }
+
+  .text {
+    width: 100%;
+    max-width: 750px;
+    height: 600px;
+  }
+
+  .vectormob {
+    position: absolute;
+    width: 67px;
+    height: 39px;
+    left: 345px;
+    top: 1016px;
+    display: block;
+  }
+}
+
+@media (max-width: 660px) {
+  .first-container {
+    padding: 267px 0;
+    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
+    background-size: 660px;
+    background-position: center;
+  }
+
+  .header__inner {
+    width: 100%;
+    height: 123px;
+    background: #fff;
+    z-index: 2;
+  }
+
+  .header__menu {
+    background: #EDEDED;
```

Comment:
Codacy found an issue: [Expected "#EDEDED" to be "#ededed" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)